Neos.Neos/Classes/Utility/NodeAddressNormalizer.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Neos\Neos\Utility;
+
+use Neos\ContentRepository\Core\Projection\ContentGraph\AbsoluteNodePath;
+use Neos\ContentRepository\Core\Projection\ContentGraph\Node;
+use Neos\ContentRepository\Core\Projection\ContentGraph\NodePath;
+use Neos\ContentRepository\Core\SharedModel\Node\NodeAddress;
+use Neos\ContentRepositoryRegistry\ContentRepositoryRegistry;
+use Neos\Flow\Annotations as Flow;
+
+final class NodeAddressNormalizer
```

Comment:
I think this is the wrong name still ... maybe `NodePathResolver::resolveNodeAddressFromPath` or `NodePathUtitlity::retrieveNodeAddressByPath`  idont know ...
or does *Normalizer fit the usecase here?

cc @kitsunet

Reply:
Jup, added internal and renamed it to `NodePathResolver::resolveNodeAddressByPath` ... the normalise naming was stolen from 8.3 but doesnt fit anymore ^^